suite-native/navigation/src/components/ScreenHeader.tsx

```diff
@@ -61,7 +68,15 @@ export const ScreenHeader = ({
             ) : (
                 <View style={applyStyle(iconWrapperStyle)}>{leftIcon}</View>
             )}
-            {title ? <Text variant="titleSmall">{title}</Text> : titleComponent}
+            {title && titleComponent ? (
+                <Box style={applyStyle(titleStyle)}>
+                    {titleComponent}
```

Comment:
This is very confusing, why would you allow rendering both `titleComponent` and `title`? I don't think we will need this that much flexible and for better readability I would prefer to rename this or `progressBarComponent` or maybe even better hardcode progress bar here and show it only when you pass some progress bar config props (`activeStep`, `numberOfSteps`)

Reply:
f70c4faa99a5dd7598596b6b7a30aefbe99ac910